charts/agent-group/values.schema.yaml

```diff
@@ -108,3 +109,6 @@ properties:
   env:
     description: List of additional environment variables that may be specified in the container
     $ref: "#/definitions/env"
+  tolerations:
+    description: Pod tolerations to node tains
```

Comment:
Typo?

Reply:
@tmercswims I think it is correct, it should be `tolerations` or maybe I am not spotting the right word where I might have made a typo.